src/components/list/list.ts

```diff
@@ -0,0 +1,56 @@
+import {Component, ViewEncapsulation, ContentChildren, ContentChild, QueryList, Directive,
+  ElementRef, Renderer} from 'angular2/core';
+import {CONST_EXPR} from 'angular2/src/facade/lang';
+
+@Component({
+  selector: 'md-list',
+  host: {'role': 'list'},
+  template: '<ng-content></ng-content>',
+  styleUrls: ['./components/list/list.css'],
+  encapsulation: ViewEncapsulation.None
+})
+export class MdList {}
+
+@Directive({ selector: '[md-line]' })
```

Comment:
Add comments explaining thay you define these directives so that you can query for them with `@ContentChildren`.